test/test.js

```diff
@@ -258,7 +258,7 @@ describe('Testing APIs', function() {
                     .send(updatedStatus)
                     .expect(200)
                     .end(function(err, res) {
-                        res.body.info.should.equal('PCS003 : Successfully updated user progress status');
+                        res.body.info.should.equal('Successfully updated user progress status');
```

Comment:
Similarly these changes can be taken up in another PR

Reply:
yes..